src/ResourceManager/SiteRecovery/Commands.SiteRecovery/Network/RemoveAzureRMSiteRecoveryNetworkMapping.cs

```diff
@@ -22,7 +22,7 @@ namespace Microsoft.Azure.Commands.SiteRecovery
     /// </summary>
     [Cmdlet(VerbsCommon.Remove, "AzureRmSiteRecoveryNetworkMapping")]
     [OutputType(typeof(ASRJob))]
-    public class RemoveAzureRMSiteRecoveryNetworkMapping : SiteRecoveryCmdletBase
```

Comment:
We get errors from the build infra for SupportShouldProcess parameter only if cmdlet has -Force parameter.  I guess if Force is not present SupportShouldProcess is not mandatory as PR build succeeds.

---
In reply to: [78791145](https://github.com/Azure/azure-powershell/pull/2912#discussion_r78791145) [](ancestors = 78791145)